src/Search/Microsoft.Azure.Search/project.json

```diff
@@ -1,57 +1,84 @@
 {
-  "version": "2.0.0-preview",
+  "version": "3.0.0-preview",
```

Comment:
Please do not increment the major version number for Search. We have not shipped any 2.x versions to NuGet yet. Also please do not publish this package as it is not ready to ship yet. We are currently doing feature development in another branch (search-v2-preview-dev).